python/kvikio/tests/test_http_io.py

```diff
@@ -18,6 +22,58 @@
 )
 
 
+class ErrorCounter:
+    # ThreadedHTTPServer creates a new handler per request.
+    # This lets us share some state between requests.
+    def __init__(self):
+        self.value = 0
+
+
+class HTTP500Handler(SimpleHTTPRequestHandler):
```

Comment:
This appears to be issuing 503, not 500.
```suggestion
class HTTP503Handler(SimpleHTTPRequestHandler):
```